### PR TITLE
CircleCI config init

### PR DESCRIPTION
**Pull request by DandyCV**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      name: Installing gems
+      command: bundle install --path vendor/bundle
+
+  save_bundle_cache: &save_bundle_cache
+    save_cache:
+      key: todo-api-{{ checksum "Gemfile.lock" }}
+      paths:
+        - vendor/bundle
+
+version: 1
```
Comment (bestwebua) [inline .circleci/config.yml]: ???

**Review by bestwebua — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - image: circleci/postgres:12.6
+      environment:
+        POSTGRES_USER: root
+        POSTGRES_PASSWORD: ""
+
+references:
+  install_bundler: &install_bundler
+    run:
+      name: Installing bundler
+      command: gem i bundler -v 2.2.18
```
Comment (bestwebua) [inline .circleci/config.yml]: Don't hardcode it.
```
gem i bundler -v $(tail -1 Gemfile.lock | tr -d ' ')
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      - store_artifacts:
+          path: ~/planner-api/coverage
+          destination: coverage
+
+workflows:
+  version: 1
+  build:
+    jobs:
+      - linters
+      - tests
```
Comment (bestwebua) [inline .circleci/config.yml]: End of line

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,113 @@
+defaults: &defaults
+  working_directory: ~/Planner-API
```
Comment (bestwebua) [inline .circleci/config.yml]: ~/planner-api

**Review by bestwebua — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (DandyCV) [reply .circleci/config.yml]: Added

File: .circleci/config.yml (hunk 4 above)
Comment (DandyCV) [reply .circleci/config.yml]: fixed

File: spec/examples.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+example_id                                    | status | run_time        |
```
Comment (bestwebua) [inline spec/examples.txt]: To gitignore

File: .rubocop.yml [hunk 6]
```diff
@@ -100,3 +101,134 @@ RSpec/InstanceVariable:
...
+Rails/SquishedSQLHeredocs: # (new in 2.8)
+  Enabled: true
+Rails/TimeZoneAssignment: # (new in 2.10)
+  Enabled: true
+Rails/WhereEquals: # (new in 2.9)
+  Enabled: true
+Rails/WhereExists: # (new in 2.7)
+  Enabled: true
+Rails/WhereNot: # (new in 2.8)
+  Enabled: true
```
Comment (bestwebua) [inline .rubocop.yml]: End of line

**Review by bestwebua — CHANGES_REQUESTED** [review]

**Review by bestwebua — APPROVED** [review]
👍

**Pull request merged**